MeesGame/MeesGame.csproj

```diff
@@ -119,6 +119,7 @@
     <Compile Include="UI\Controls\RadioButton.cs" />
     <Compile Include="UI\Controls\SpriteSheetButton.cs" />
     <Compile Include="UI\Controls\TextureButton.cs" />
+    <Compile Include="UI\DefaultUIValues.cs" />
```

Comment:
This file causes errors as it doesn't exist. If you did mean to add this file, add the actual file. If you didn't mean to, remove it from the csproj